### PR TITLE
Update ebpf_tools_in_minikube.md

### PR DESCRIPTION
**Pull request by medyagh**
bucket doesnt exist anymore and no longer needed to have custom ISO
https://github.com/kubernetes/minikube/issues/19886

### CODE REVIEW

File: site/content/en/docs/tutorials/ebpf_tools_in_minikube.md [hunk 1]
```diff
@@ -13,29 +13,21 @@ eBPF tools are performance tools used for observing the Linux kernel.
 These tools can be used to monitor your Kubernetes application in minikube.
 This tutorial will cover how to set up your minikube cluster so that you can run eBPF tools from a Docker container within minikube.
 
+
+## Requirment
```
Comment (spowelljr) [inline site/content/en/docs/tutorials/ebpf_tools_in_minikube.md:17]:
```suggestion
## Requirement
```

File: site/content/en/docs/tutorials/ebpf_tools_in_minikube.md [hunk 2]
```diff
@@ -13,29 +13,21 @@ eBPF tools are performance tools used for observing the Linux kernel.
...
+- x86 (currently bcc image does not support arm64)
```
Comment (spowelljr) [inline site/content/en/docs/tutorials/ebpf_tools_in_minikube.md:18]:
```suggestion
- x86 (currently the [bcc image](https://hub.docker.com/r/zlim/bcc/tags) does not support arm64)
```

File: site/content/en/docs/tutorials/ebpf_tools_in_minikube.md [hunk 3]
```diff
@@ -13,29 +13,21 @@ eBPF tools are performance tools used for observing the Linux kernel.
...
 ## Prerequisites
 
 - Latest minikube binary
```
Comment (spowelljr) [inline site/content/en/docs/tutorials/ebpf_tools_in_minikube.md:21]: We can probably update this to the version after the PR was merged to add eBPF to the ISO

File: site/content/en/docs/tutorials/ebpf_tools_in_minikube.md (hunk 2 above)
Comment (spowelljr) [inline site/content/en/docs/tutorials/ebpf_tools_in_minikube.md:18]: This should just be added to `Prerequisites` below, they're the same thing

File: site/content/en/docs/tutorials/ebpf_tools_in_minikube.md (hunk 3 above)
Comment (spowelljr) [inline site/content/en/docs/tutorials/ebpf_tools_in_minikube.md:21]: We should add VM driver here as well

Comment (medyagh) [reply site/content/en/docs/tutorials/ebpf_tools_in_minikube.md]: done ! PTAL

**Review by spowelljr — APPROVED** [review]

**Pull request merged**